app/validators/profile_validator.rb

```diff
@@ -32,19 +32,10 @@ def validate(record)
   private
 
   def summary_too_long?(record)
-    # Calling the summary attribute method on record during onboarding
-    # throws a NoMethodError
-    return unless record.respond_to?(SUMMARY_ATTRIBUTE)
-
-    # TODO: [@jacobherrington] This will need to be addressed when the summary
-    # ProfileField is dropped from production.
-    return unless ProfileField.exists?(attribute_name: SUMMARY_ATTRIBUTE)
```

Comment:
We don't need this anymore since `summary` is now an actual column, not a `ProfileField`.